test/test_script.py

```diff
@@ -1568,3 +1568,35 @@ def test_cmd_unique(self):
             self.parser.eval("$unique()")
         with self.assertRaisesRegex(ScriptError, areg):
             self.parser.eval("$unique(B:AB; D:C; E:D; A:A; C:X,1,:,extra)")
+
+    def test_cmd_countryname(self):
+        from picard import i18n
+        context = Metadata()
+        context["foo"] = "ca"
+        context["bar"] = ""
+        context["baz"] = "INVALID"
+
+        # Test with Russian locale (Disabled because test fails on GitHub actions. Perhaps locale information is not available?)
```

Comment:
Yes, there's no sync for translation files in the workflow.
That said, it's a bad idea to rely on external files for tests (as values can change, unlikely in this case but still).

Instead you should override underlying `gettext_countries()` to return a constant value:

```python
        def mock_gettext_countries(arg):
            return "Канада"

        from unittest import mock
        with mock.patch('builtins.gettext_countries', mock_gettext_countries):
            self.assertScriptResultEquals("$countryname(ca)", "Canada", context)
            self.assertScriptResultEquals("$countryname(ca,)", "Canada", context)
            self.assertScriptResultEquals("$countryname(ca, )", "Канада", context)
            self.assertScriptResultEquals("$countryname(ca,yes)", "Канада", context)
```